todo_app/tests/test_viewmodel.py

```diff
@@ -0,0 +1,30 @@
+from todo_app.viewmodels import IndexViewModel
+from todo_app.data.trello_items import Item, List
+
+test_items = [
+    {"id": 1, "name": "Task 1", "desc": "The first task", "due": ""},
+    {"id": 2, "name": "Task 2", "desc": "The second task", "due": ""},
+    {"id": 3, "name": "Task 3", "desc": "The third task", "due": ""},
+    {"id": 4, "name": "Task 4", "desc": "The fourth task", "due": ""}
+]
+
+
+def test_viewmodel_todo_property():
+    test_lists = [List(1, "To Do", test_items)]
+    viewmodel = IndexViewModel(test_lists)
+
+    assert len(viewmodel.to_do_items) == 4
```

Comment:
If these were real tests I'd definitely want to see some non-"To Do" data in here as well to check it is actually filtering vaguely sensibly!